.github/workflows/release.yaml

```diff
@@ -33,19 +34,17 @@ jobs:
       - name: Get release version
         id: release-version
         run: |
-            if [[ ${{ steps.changed-files.outputs.all_changed_files_count }} != 1 ]]; then
-              echo "1 release notes file should be changed to create a release tag, found ${{ steps.changed-files.outputs.all_changed_files_count }}"
-              exit 1
-            fi
             for changed_file in ${{ steps.changed-files.outputs.all_changed_files }}; do
-              export RELEASE_VERSION=$(echo "${changed_file}" | grep -oP '(?<=/)[^/]+(?=\.md)')
-              echo "RELEASE_VERSION=$RELEASE_VERSION" >> $GITHUB_ENV
-              echo "RELEASE_VERSION=$RELEASE_VERSION" >> $GITHUB_OUTPUT
-              if [[ "$RELEASE_VERSION" =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$ ]]; then
-                echo "Valid semver: $RELEASE_VERSION"
-              else
-                echo "Invalid semver: $RELEASE_VERSION"
-                exit 1
+              if [[ $changed_file =~ CHANGELOG/.*\.md ]]; then
```

Comment:
How do we expect this to behave when multiple CHANGELOG files are updated at the same time?

Reply:
Is it possible to modify the `changed-files` action to only look at CHANGELOG files?

Reply:
We expect it to misbehave in that case I guess: it would only process the first CHANGELOG.md file found. @dtzar and I thought this was unlikely, but this could be replaced with a proper loop if prefererred.

Reply:
> Is it possible to modify the changed-files action to only look at CHANGELOG files?

Yes, according to the README https://github.com/tj-actions/changed-files?tab=readme-ov-file#usage-
```yaml
      - name: Get all changed markdown files
        id: changed-markdown-files
        uses: tj-actions/changed-files@v42
        with:
          # Avoid using single or double quotes for multiline patterns
          files: |
             **.md
```